### DISCUSSION TITLE
Style binding Typescript

### DISCUSSION THREAD

**Discussion by mitraecp** [nuxt/nuxt#17027] [Questions]
How can i set styles dynamic with typescript?

![image](https://user-images.githubusercontent.com/75841640/186001531-8993b799-73ef-4e1b-a2eb-226b6e22d352.png)

**Comment by JiatLn** [comment]
may you could try to add this config in `tsconfig.json`.
```ts
{
  // https://v3.nuxtjs.org/concepts/typescript
  "extends": "./.nuxt/tsconfig.json",
  // ↓ add this option
  "vueCompilerOptions": {
    "experimentalDisableTemplateSupport": true
  }
}
```
